recipes/sqls/conda_build_config.yaml

```
@@ -0,0 +1,4 @@
+c_stdlib_version:  # [osx]
+  - "10.15"  # [osx]
+MACOSX_SDK_VERSION:  # [osx]
+  - "10.15"  # [osx]
```

This should not apply to `osx-arm64`
```suggestion
c_stdlib_version:  # [osx and x86_64]
  - "10.15"  # [osx and x86_64]
MACOSX_SDK_VERSION:  # [osx and x86_64]
  - "10.15"  # [osx and x86_64]
```